### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hi,

Some minor fixes

Cheers,
Marcus

### CODE REVIEW

File: mxcube3/video/streaming.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def new_frame_received(img, width, height, *args, **kwargs):
         # start the streaming process if not started or restart if terminated
         if not VIDEO_STREAM_PROCESS or VIDEO_STREAM_PROCESS.poll() is not None:
             sfpath = os.path.join(os.path.dirname(__file__), "streaming_processes.py")
-            VIDEO_STREAM_PROCESS = subprocess.Popen([sys.executable, sfpath, VIDEO_DEVICE.name, VIDEO_SIZE])
+            VIDEO_STREAM_PROCESS = subprocess.Popen(["python", sfpath, VIDEO_DEVICE.name, VIDEO_SIZE])
```
Comment (mguijarr) [inline mxcube3/video/streaming.py]: It seems like the more portable and reliable way is to use a trick like this:

```import os
python_executable = os.sep.join(os.path.dirname(os.__file__).split(os.sep)[:-2]+["bin", "python"])
```
I am afraid using just "python" could lead to surprising results when virtual envs are used, but maybe not...

Comment (marcus-oscarsson) [reply mxcube3/video/streaming.py]: I dont mind using this fix, but if the problem is that "python" does not point to the correct version of python within a virtual environment I would rather consider it as a problem with that virtual environment

**Comment by meguiraun** [comment]
thanks to both! @mguijarr  good to see you around ;)

**Comment by mguijarr** [comment]
@meguiraun thanks ;) I am turning into a jedi master "force ghost" for MXCuBE ;)

**Pull request merged**